app/api/auth/[...nextauth]/route.ts

```diff
@@ -20,25 +21,27 @@ const authOptions: NextAuthOptions = {
     CredentialsProvider({
       type: "credentials",
       credentials: {
-        email: {},
-        password: {},
+        code: {},
       },
-      async authorize(credentials, req) {
-        if (!credentials) throw new Error("No credentials found!");
-        const { email, password } = credentials;
-
+      async authorize(credentials) {
+        const code = credentials?.code;
+        if (!code) {
+          throw new Error("Missing code");
```

Comment:
Maybe I wasn't clear about what I was suggesting.  Would anything break if we just reworded the error message to something like this:
```suggestion
          throw new Error(`Credentials missing required "code" property: ${credentials}`);
```
?

Reply:
> Maybe I wasn't clear about what I was suggesting. Would anything break if we just reworded the error message to something like this:
 Got it. Looks good to me. Thanks for the suggestion!